tests/integration_tests/functional/test_serial_io.py

```diff
@@ -44,6 +46,56 @@ def handle_input(self, unused_serial, _) -> TestState:
         return self
 
 
+def test_serial_after_snapshot(bin_cloner_path):
+    """
+    Test that writing to stdout works after restore.
```

Comment:
you cannot write to stdout 😄 
I suggest:
```suggestion
    Test serial I/O after restoring from a snapshot.
```